Makefile

```diff
@@ -35,7 +40,7 @@ check-fuzz-%: tests/fuzz-% FORCE
 fuzz-check: check-fuzz-ppm check-fuzz-jpg check-fuzz-header check-fuzz-text check-fuzz-dstr
 
 format: FORCE
-	$(CLANG_FORMAT) -i pdfgen.c pdfgen.h tests/main.c tests/fuzz-ppm.c tests/fuzz-jpg.c tests/fuzz-header.c tests/fuzz-text.c
+	$(CLANG_FORMAT) -i pdfgen.c pdfgen.h tests/main.c tests/fuzz-ppm.c tests/fuzz-jpg.c tests/fuzz-header.c tests/fuzz-text.c tests/penguin.c
```

Comment:
We don't need to format this new file, since it's autogenerated.

Reply:
Done.